jbpm-db-scripts/src/main/resources/db/upgrade-scripts/derby/jbpm-7.55-to-7.56.sql

```diff
@@ -0,0 +1,7 @@
+alter table AuditTaskImpl add column end_date timestamp;
```

Comment:
Filename shouldn't be `jbpm-7.49-to-7.56.sql`?

Reply:
nope... the change is between those versions